### PR TITLE
Add Devise

### PR DESCRIPTION
**Pull request by rachidelaid**
> tasks done:
- installed and set upped devise.
- generated and modified the devise views.
- added `letter-opener` to see the emails.
- styled the forms.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,5 +1,17 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  # def current_user
+  #   User.first
+  # end
```
Comment (elyor-doniyorov) [inline app/controllers/application_controller.rb:4]: - [ ] Kindly remove unused code from your code base in order to keep it clean.

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <% posts_counter: 0 %>
+
+  <div class="field">
+    <%= f.label :name %>
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
```
Comment (elyor-doniyorov) [inline app/views/devise/registrations/new.html.erb]: - [ ] I keep getting the following error when try to sign up. :point_down: 
Screenshot from 2022-05-09 22-07-45
Please fix it.

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @rachidelaid, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: There are no linter errors
:heavy_check_mark: Devise has been installed and setup by the student using the [getting started link](https://github.com/heartcombo/devise).
:heavy_check_mark: etc.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello team :wave: , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**